.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,10 +18,10 @@ if the PR resolves an issue, please add further checkboxes that describe the
 action items or test scenarios from it
 -->
 
-- [ ] I have read the **CONTRIBUTING** document
-- [ ] my code follows the style requirements
+- [ ] I have read the [CONTRIBUTING.md](https://github.com/tomato42/tlsfuzzer/blob/master/CONTRIBUTING.md) document
+- [ ] my code follows the [PEP 8](https://www.python.org/dev/peps/pep-0008/) and [PEP 257](https://www.python.org/dev/peps/pep-0257/) style guides
 - [ ] the changes are also reflected in documentation and code comments
-- [ ] the added test cases cover the modified or added code
+- [ ] the added test cases cover the modified or added code (I have checked coveralls.io report)
```

Comment:
Overall I find the tickbox quite large to be practical. It may introduce a significant barrier to first commit and some of the items look like documentation in CONTRIBUTION.md

Reply:
yeah, unfortunately there are multiple things to be mindful when introducing changes, and I prefer to have it in writing